org.zim_wiki.Zim.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id": "org.zim_wiki.Zim",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zim",
+    "rename-desktop-file": "zim.desktop",
+    "rename-icon": "zim",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        /* libappindicator access */
+        "--socket=session-bus",
```

Comment:
You should only need a restricted set of permissions here rather than this general call. e.g. https://github.com/flathub/io.github.Pithos/blob/master/io.github.Pithos.json#L35 is an example of an app with appindicator integration.

Reply:
`org.kde.StatusNotifierWatcher` is the modern bus name.